### PR TITLE
feat(app-shell): to use all feature toggles query

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

This pull request builds on a new query in the http proxy `allFeatureToggles`.

#### Description

This query will return all toggls configured in both nav and app menu. This data we can then use to send it down to flopflip.

### CODE REVIEW

File: packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { useQuery } from 'react-apollo';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { FetchAllMenuFeatureToggles } from './fetch-all-menu-feature-toggles.proxy.graphql';
```
Comment (emmenko) [inline packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js]: Don't forget to pull the schema before merging

Comment (tdeekens) [reply packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js]: Yup. Just need it deployed :). Will continue next week.

**Comment by tdeekens** [comment]
@emmenko ready for review. I qualified it as a bug as I am not sure it can be seen as a feature.

**Comment by tdeekens** [comment]
Once approved I will deploy the mc proxy and pull the schemas.

File: packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js (hunk 1 above)
Comment (emmenko) [reply packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js]: Did you pull the schema?

Comment (tdeekens) [reply packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.js]: CleanShot 2019-11-11 at 12 54 55@2x

File: packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.spec.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      await waitForElement(() => rendered.getByText('Loading'));
+      await waitForElement(() => [
+        rendered.getByText(/Number of toggles: 2/i),
+        rendered.getByText(/Toggle: flagA/i),
+        rendered.getByText(/Toggle: flagB/i),
+      ]);
+    });
+  });
+
+  describe('with error', () => {
```
Comment (emmenko) [inline packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.spec.js]: I think we can remove this test, it’s not really super important. It has also been kind of flaky in the past, not sure if with the more recent versions of apollo it got more stable or not.

Comment (tdeekens) [reply packages/application-shell/src/hooks/use-all-menu-feature-toggles/use-all-menu-feature-toggles.spec.js]: Hm, thanks. I'll keep it and see how stable it'll be. The ones in the MC for signup etc include loads of error states and are quite stable.

**Pull request merged**